README.md

```diff
@@ -81,30 +81,26 @@ export default handleAuth();
 
 This will create the following urls: `/api/auth/login`, `/api/auth/callback`, `/api/auth/logout` and `/api/auth/me`.
 
-Wrap your `pages/_app.jsx` component in the `UserProvider` component.
+Wrap your `pages/_app.js` component in the `UserProvider` component.
 
-```jsx
-// pages/_app.jsx
+```js
+// pages/_app.js
```

Comment:
Renamed these extensions as it seems `.js` is more common, and it's what you get out of the box with a new Next app today.